### PR TITLE
fix: HS-175: Improved prompts and endpoint fix

### PR DESCRIPTION
**Pull request by ArushKapoorJuspay**
**Problem Description -** OSS setup improvement

**Solution -**

- [x] Asking for prompts while running `npm run start:playground` only when they are empty
- [x] Make the  playground prompts more descriptive and understandable as well as add the explanation in the README for the configs

### CODE REVIEW

File: Hyperswitch-React-Demo-App/promptScript.js [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
 const envPath = "./.env";
 
 const obj = {
-  HYPERSWITCH_PUBLISHABLE_KEY: publishableKey,
-  HYPERSWITCH_SECRET_KEY: secretKey,
-  HYPERSWITCH_SERVER_URL: serverURL,
-  HYPERSWITCH_CLIENT_URL: clientURL,
-  SELF_SERVER_URL: appServerURL,
-  SELF_CLIENT_URL: appClientURL,
+  HYPERSWITCH_PUBLISHABLE_KEY: "Publishable Key",
```
Comment (prafulkoppalkar) [inline Hyperswitch-React-Demo-App/promptScript.js:7]: if you are having prompts only when the env is empty you might as well initialize these to empty string 
just to make sure that script doesnt update the value with these

File: Hyperswitch-React-Demo-App/webpack.dev.js [hunk 2]
```diff
@@ -8,11 +8,11 @@ let devServer = {
   port: 9060,
   historyApiFallback: true,
   proxy: {
-    "/payments": {
+    "/": {
```
Comment (prafulkoppalkar) [inline Hyperswitch-React-Demo-App/webpack.dev.js:11]: please remove this change cuz we have already fixed this and tested with docker setup
take a look at this PR for your reference

**Review by prafulkoppalkar — CHANGES_REQUESTED** [review]

File: Hyperswitch-React-Demo-App/webpack.dev.js (hunk 2 above)
Comment (ArushKapoorJuspay) [reply Hyperswitch-React-Demo-App/webpack.dev.js:11]: image

The PR does not contain the change in the proxy. It is still hitting /payments, will that not cause any issue?

File: Hyperswitch-React-Demo-App/promptScript.js (hunk 1 above)
Comment (ArushKapoorJuspay) [reply Hyperswitch-React-Demo-App/promptScript.js:7]: These values are not what will be added in the env. These are the string for the prompt messages. I have moved the prompt inside the initializeValues function

File: Hyperswitch-React-Demo-App/webpack.dev.js (hunk 2 above)
Comment (arun-mi) [reply Hyperswitch-React-Demo-App/webpack.dev.js:11]: It has to be /payments in both places, we had asked for the change to be made in  the CDK script itself.

Comment (ArushKapoorJuspay) [reply Hyperswitch-React-Demo-App/webpack.dev.js:11]: Okay, will revert this change

File: Hyperswitch-React-Demo-App/promptScript.js (hunk 1 above)
Comment (prafulkoppalkar) [reply Hyperswitch-React-Demo-App/promptScript.js:7]: got it

**Review by prafulkoppalkar — APPROVED** [review]

**Pull request merged**